main/app/contextualLayer/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright: (C) 2015 WYSIWYD Consortium
+# Authors: Clement Moulin-Frier
+# CopyPolicy: Released under the terms of the GNU GPL v2.0.
+
+# add_subdirectory(goalManager)
```

Comment:
that's because `goalManager` is not being used. I kept it in the source because I foresee there might be a need for a second module that operates like the planner but for more specific goals.